src/wallet/rpcwallet.cpp

```diff
@@ -2404,7 +2408,7 @@ UniValue getwalletinfo(const UniValue& params, bool fHelp)
     obj.pushKV("walletversion", pwalletMain->GetVersion());
     obj.pushKV("balance",       ValueFromAmount(pwalletMain->GetBalance(asOfHeight)));
     if (!asOfHeight.has_value()) {
-        obj.pushKV("unconfirmed_balance", ValueFromAmount(pwalletMain->GetUnconfirmedTransparentBalance()));
+        obj.pushKV("unconfirmed_balance", ValueFromAmount(pwalletMain->GetUnconfirmedTransparentBalance(ISMINE_SPENDABLE)));
```

Comment:
Should the balances in `getwalletinfo` be restricted to only those values not associated with unified accounts? Or do we want the totals here to be representative of the wallet as a whole?

Reply:
The current RPC doc for `getwalletinfo` would suggest the wallet as a whole (but it's worth putting an explicit note about this in the doc).